src/common/utils/validationRules.ts

```diff
@@ -11,6 +11,9 @@ export default function validate(values: validateProps) {
   } else if (!/\S+@\S+\.\S+/.test(values.email)) {
     errors.email = "Email address is invalid";

```

Comment:
- **Runtime error check**  
  The `errors` object is not checked for null or undefined before being used. This could lead to a runtime error if `errors` is null or undefined.